<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Sair</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(9, 38, 66);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 118, 209);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string>SAIR DO WINDOWS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgba(255, 255, 255, 254);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Em quanto tempo deseja&lt;br/&gt;sair do windows?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(128, 185, 238, 50);
border-radius: 20px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="quadro_pri">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>37</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 90pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>01</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="quadro_sec">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>37</y>
       <width>131</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 90pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>47</y>
       <width>33</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font: 90pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font:20pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Horas</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>131</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font:20pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Minutos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="up_pri">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>12</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font-size:35px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>⮝</string>
     </property>
    </widget>
    <widget class="QPushButton" name="up_sec">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>12</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font-size:35px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>⮝</string>
     </property>
    </widget>
    <widget class="QPushButton" name="down_pri">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>147</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font-size:35px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>⮟</string>
     </property>
    </widget>
    <widget class="QPushButton" name="down_sec">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>147</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
font-size:35px;
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>⮟</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="modo">
    <property name="geometry">
     <rect>
      <x>228</x>
      <y>395</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 118, 209);
border-radius: 5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
    <item>
     <property name="text">
      <string>Desligar</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Reiniciar</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_iniciar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 118, 209);
border-radius: 10px;
color: rgb(255, 255, 255);
font: 700 12pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
